<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gallery</class>
 <widget class="QWidget" name="Gallery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Галерея</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gallery_gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="add_pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>358</width>
         <height>357</height>
        </rect>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>321</width>
          <height>351</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="image_gridLayout"/>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="delete_pushButton">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="back_pushButton">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
